cli/index.ts

```diff
@@ -439,6 +439,22 @@ function handleError(error: Error) {
 function getObjectOrJSON(options) {
   switch (typeof options) {
     case 'object':
+      if (options.versionData) {
```

Comment:
NIT: It might be worth doing optional chaining here since the `typeof null` is also `'object'`, one of JavaScript's weird quirks.

example:
`if (options?.versionData)`

Reply:
good eye!